src/DomainObjects/Repository.cs

```diff
@@ -6,11 +6,14 @@ namespace DomainObjects
     public class Repository : IEntity
     {
         public Guid Id { get; set; }
+        public string Name { get; set; }
         public DateTime Created { get; set; }
         public DateTime Updated { get; set; }
 
         public Guid OrganizationId { get; set; }
-        public string Name { get; set; }
+        public string OrganizationName { get; set; }
```

Comment:
Why denormalize this, instead of fetching it from the Organization itself? If it's not actually the Organization name then this should probably go in "CustomRepositoryInformation".

Reply:
My mistake. I didn't realize that repository wasn't like a git repository